### PR TITLE
Android dev ws

### PR DESCRIPTION
**Pull request by oykuyilmaz1**
Please check how edit, add, and delete workspace are working.

### CODE REVIEW

File: Platon/app/src/main/java/com/cmpe451/platon/adapter/WorkspaceListAdapter.kt [hunk 1]
```diff
@@ -85,6 +85,7 @@ class WorkspaceListAdapter(private val data: ArrayList<WorkspaceListItem>, priva
         this.notifyDataSetChanged()
     }
     fun submitElements(list: List<WorkspaceListItem>){
+        data.clear()
```
Comment (darktheorys) [inline Platon/app/src/main/java/com/cmpe451/platon/adapter/WorkspaceListAdapter.kt:88]: i remember, data.clear() prevents pagination at some respect, can you double check.

Comment (oykuyilmaz1) [reply Platon/app/src/main/java/com/cmpe451/platon/adapter/WorkspaceListAdapter.kt:88]: we do not have pagination for workspaces, so it was doubling the data

File: Platon/app/src/main/res/drawable/ic_delete_outline_white_24dp.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:pathData="M6,19c0,1.1 0.9,2 2,2h8c1.1,0 2,-0.9 2,-2L18,7L6,7v12zM8,9h8v10L8,19L8,9zM15.5,4l-1,-1h-5l-1,1L5,4v2h14L19,4z"
+      android:fillColor="#ffffff"/>
```
Comment (darktheorys) [inline Platon/app/src/main/res/drawable/ic_delete_outline_white_24dp.xml:8]: it is black, not our primary or secondary color, do double check please

**Review by darktheorys — APPROVED** [review]

File: Platon/app/src/main/res/drawable/ic_delete_outline_white_24dp.xml (hunk 2 above)
Comment (oykuyilmaz1) [reply Platon/app/src/main/res/drawable/ic_delete_outline_white_24dp.xml:8]: I will change it now, thank you

**Pull request merged**